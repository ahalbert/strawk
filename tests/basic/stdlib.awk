BEGIN {
  a="123abc456xyz"
  b[1] = 1
  b[2] = 2
  b[3] = 3
  print a, length(a)
  print length(b)
  print sub(/abc/, "789", a)
  c="123abc456xyzabc"
  print gsub(/abc/, "789", c)
}
