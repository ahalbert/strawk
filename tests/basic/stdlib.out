123abc456xyz 12
3
123789456xyz
123789456xyz789
